File: C/primeno.c
#include <stdio.h>
main() {
	int x,i;
	int count = 0;
	scanf("%d",&x);
	for ( i = 2;i < x;i++) {
		if (x%i == 0) {
			break;
		} 	printf("%d is not a prime no",x);
		else {
			printf("%d is a prime no",x);
		}
	}
	


}

File: JAVA/java pograme/primeno.java
import java.util.*; // contains Collections framework

// don't change the name of this class
// you can add inner classes if needed
class primeno {
    public static void main (String[] args) {
        int count = 0;
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        for (int i = 1;i<=a;i++) {
            if (a % i == 0) {
                count++;
            }
        }
            if (count == 2) {
                System.out.println("This is a Prime Number");
            } else {
                System.out.println("This is Not a Prime Number");
            }
        }

    }

